docs/platform-external/installing.md

```diff
@@ -0,0 +1,509 @@
+      [Unit]
+      Description=Fetch kubelet provider id from Metadata
+      # Run afterburn service for collect info from metadata server
+      # see: https://coreos.github.io/afterburn/usage/attributes/
+      # Not required due to OCP 4.1 boot image does not contain afterburn service
+      # see: https://issues.redhat.com/browse/OCPBUGS-7559
+      Wants=afterburn.service
+      After=afterburn.service
+      # Wait for NetworkManager to report it's online
+      After=NetworkManager-wait-online.service
+      # Run before kubelet
+      Before=kubelet.service
+      [Service]
+      # Mark afterburn environment file optional, due to it is possible that afterburn service was not executed
+      EnvironmentFile=-/run/metadata/afterburn
+      ExecStart=/usr/local/bin/kubelet-providerid
+      Type=oneshot
+      [Install]
+      WantedBy=network-online.target
+```
+
+The `{{ machine_role }}` must be `master` and `worker`. Both manifests must be
+saved to `config-master.bu` and `config-worker.bu`, then run the following commands:
+
+```bash
+$ butane config-master.bu \
+  -o openshift/99_openshift-machineconfig_00-master-kubelet-providerid.yaml
+
+$ butane config-worker.bu \
+  -o openshift/99_openshift-machineconfig_00-worker-kubelet-providerid.yaml
+```
+
+#### Create additional manifests
+
+If the CCM or any other component you want to add in install time requires
+dependencies, you can also add it to the respective directories.
+
+For example, if the CCM requires secrets or custom configuration, you can create it:
+
+> replace the `{{ secret_value | b64encode }}` to a valid base64 string
+
+```bash
+cat <<EOF > manifests/cloud-controller-manager-01-secret.yaml
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cloud-controller-manager-config
+data:
+  cloud-provider.yaml: {{ secret_value | b64encode }}
+EOF
+```
+
+### Create ignition files
+
+Once the manifests are placed, you can create the cluster ignition configurations:
+
+~~~bash
+./openshift-install create ignition-configs
+~~~
+
+The files with the extension `.ign` must be generated.
+
+```text
+.
+├── auth
+│   ├── kubeadmin-password
+│   └── kubeconfig
+├── bootstrap.ign
+├── master.ign
+├── metadata.json
+└── worker.ign
+```
+
+## Create compute nodes
+
+The step to provision compute nodes must be cloud-specific. The required steps
+to boot RHCOS is different for each compute role:
+
+- bootstrap node: must use the ignition file `bootstrap.ign`
+- control-plane nodes: must use the ignition file `master.ign`
+- compute nodes: must use the ignition file `worker.ign`
+
+Requirements:
+
+- You created the ignition configs
+- You uploaded the RHCOS image to the cloud image service
+- You reviewed the ["Minimum resource requirements for cluster installation"](https://docs.openshift.com/container-platform/4.13/installing/installing_platform_agnostic/installing-platform-agnostic.html#installation-minimum-resource-requirements_installing-platform-agnostic)
+
+### Bootstrap
+
```

Comment:
I'd start with a quick summary of what it is, like: "The bootstrap node is a temporary machine used only during the installation process."

Reply:
Agreed. The reader doesn't need to know implementation details, but it's helpful to have a little context for what its role is in the installation process.

Reply:
Thanks! I updated adding a quick intro.